actions/lib/compress_archive_package.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Simple script that should be executed on a biotank when we want to compress a
+# runfolder to be archived to PDC via TSM. The runfolder to archive is expected
+# to be a folder full of symlinks pointing to real source folder. 
+
+set -e
+set -o pipefail 
+
+# The full path to the runfolder that we want to archive.
```

Comment:
`compress_archive_package.sh` will never launch if the path `RUNFOLDER_NAME` doesn't contain an `Unaligned` subdir (with > `fastq_threshold` number of `.fastq.gz` files). Because the previous step in the archive workflow will then have failed. I'm not sure what we should look for here; but perhaps one could check for some expected files (e.g. a subset of the filenames in `EXCLUDE`), but I'm not completely sure if it is necessary.